darts/utils/data/tabularization.py

```diff
@@ -1243,6 +1279,10 @@ def _get_feature_times(
         [target_series, past_covariates, future_covariates],
         [lags, lags_past_covariates, lags_future_covariates],
     ):
+        # TODO: information is available in model.lags, not sure how to make the info get here
```

Comment:
can you explain a bit more what you describe here?

Reply:
Just that I need to recompute the min/max for each "variate", I was wondering if there was a nice way to pass this information to tabularization. But it's probably negligeable.